tests/e2e/compilation/test_resolution.py

```diff
@@ -43,3 +43,18 @@ def test_cycle(solc_binary_path) -> None:
     solc_path = solc_binary_path("0.8.0")
     slither = Slither(Path(TEST_DATA_DIR, "test_cyclic_import", "a.sol").as_posix(), solc=solc_path)
     _run_all_detectors(slither)
+
+
+def test_contract_function_parameter(solc_binary_path) -> None:
+    solc_path = solc_binary_path("0.8.0")
+    standard_json = SolcStandardJson()
+    standard_json.add_source_file(
+        Path(TEST_DATA_DIR, "test_contract_data", "test_contract_data.sol").as_posix()
+    )
+    compilation = CryticCompile(standard_json, solc=solc_path)
+    slither = Slither(compilation)
+    contract = slither.contracts[0]
+
+    for function in contract.functions:
+        for parameter in function.parameters:
+            str(parameter)
```

Comment:
```suggestion
        assert(function.parameters[1].name is None)
```

Reply:
For the test example, the parameter name is set to empty string: https://github.com/crytic/slither/blob/master/slither/solc_parsing/variables/variable_declaration.py#L143

Reply:
Ah, so it's not `None` it's an empty string. I guess the assertion failing was only because "" is falsy value. You can disregard my comments